VargasMartinezJose/Dis45/HackerRank/ejerciciojava.java/JavaStdinStdout1.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
```

Comment:
No using wildcards in your imports.
The default unnamed package shouldn't be used.